mmv1/products/bigquerydatatransfer/api.yaml

```diff
@@ -39,7 +39,7 @@ objects:
     references: !ruby/object:Api::Resource::ReferenceLinks
       guides:
         "Official Documentation": "https://cloud.google.com/bigquery/docs/reference/datatransfer/rest/"
-      api: "https://cloud.google.com/bigquery/docs/reference/datatransfer/rest/v1/projects.locations.transferConfigs/create"
+      api: "https://cloud.google.com/bigquery-transfer/docs/reference/datatransfer/rest/v1/projects.locations.transferConfigs"
```

Comment:
This seems to be an unrelated change; could you remove it from this PR?

Reply:
It is a doc fix. Google updated the url + the method shouldn't be in the url.

Reply:
thanks for the docs fix; since it's an unrelated change, it would be great to have it in a separate PR.

Reply:
Reverted